packages/components/src/core/ButtonIcon/index.stories.tsx

```diff
@@ -195,6 +210,246 @@ export const LivePreview = {
   render: (args: Args) => <LivePreviewDemo {...args} />,
 };
 
+// Screenshot test
+
+const ScreenshotTestDemo = (): JSX.Element => {
+  const topLevel: React.CSSProperties = {
+    columnGap: "20px",
+    display: "inline-grid",
+    fontFamily: "sans-serif",
+    marginRight: "50px",
+  };
+  const displayContents: React.CSSProperties = {
+    display: "contents",
+  };
+  const penultimateLevel: React.CSSProperties = {
+    display: "contents",
+  };
+  const bottomLevel: React.CSSProperties = {
+    marginBottom: 10,
+  };
+  const topLabel: React.CSSProperties = {
+    fontSize: "2em",
+    gridColumn: "1 / 6",
+    marginBottom: 0,
+  };
+  const midLabel: React.CSSProperties = {
+    borderStyle: "solid none none none",
+    gridColumn: "1 / 6",
+    justifySelf: "stretch",
+    paddingTop: 10,
+  };
+  const secondLabel: React.CSSProperties = {
+    ...midLabel,
+    borderWidth: "2px",
+    fontSize: "1.17em",
+    margin: "20px 0",
+  };
+  const thirdLabel: React.CSSProperties = {
+    ...midLabel,
+    alignSelf: "end",
+    borderWidth: "1px",
+    fontSize: "0.83em",
+    fontWeight: "normal",
+    margin: "0 0 5px 0",
+  };
+  const bottomLabel: React.CSSProperties = {
+    fontSize: "0.67em",
+    margin: "10px 0",
+  };
+
+  // loop through all SDS_TYPES
+  return (
+    <>
+      {SDS_TYPES.map((sdsType) => {
+        return <ButtonIconTypeOption sdsType={sdsType} key={sdsType} />;
+      })}
+    </>
+  );
+
+  // loop through all SDS_SIZES + create headers for SDS_TYPES
+  function ButtonIconTypeOption({ sdsType }: { sdsType: SDSTypes[number] }) {
+    return (
+      <div style={topLevel}>
+        <p style={topLabel}>
```

Comment:
All labels use `<p>` instead of `<h1>` now so that conditionally used labels don't break the a11y test, since headings can only increase by one to be a11y compliant

Reply:
SGTM! Thanks for the solution, Lia!